Machine learning

LucianoPC opened the pull request.

lucasmoura committed:
Filter terms using tfidf on binary vector

Signed-off-by: Lucas Moura <lucas.moura128@gmail.com>
Signed-off-by: Luciano Prestes <lucianopcbr@gmail.com>

LucianoPC committed:
Refactor flag 'option' to 'time_context'

LucianoPC committed:
Fix package classification scripts

- Add folder "user_data" on project root
- Check if files exists on src/bin/pkg_classification.py

LucianoPC committed:
Save package classification indices

LucianoPC committed:
Add src/bin/bayes_matrix.py

LucianoPC committed:
Fix pkg_classification

- Fix function 'get_pkg_data' for pass test

LucianoPC committed:
Update bayes_matrix code

LucianoPC committed:
Add bayes machine learning training

lucasmoura committed:
Remove unnecessary files from gitignore

lucasmoura committed:
Refactored bayes_matrix file

lucasmoura commented:
Luciano, o merge me parece tranquilo. Fiz uma alteração apenas no bayes_matrix.py para tornar as variáveis mais significativas. Dá uma olhada e vê se concorda. Por fim, não seria melhor trazer o bayes_matrix.py para algum arquivo do src/ ao invés do bin/ ? Pode ser dentro do próprio strategy.py ou até mesmo criar um novo módulo mesmo.

lucasmoura committed:
Fix debtags path on pkb_classification.py

LucianoPC commented:
sim, verdade, vou mandar pro src, e fazer uns testes a mais, daí eu atualizo o merge

LucianoPC committed:
Add tests for training bayes_matrix

LucianoPC committed:
Add get a classification on bayes_matrix

LucianoPC committed:
Add script to make machine learning training

- src/bin/apprec_ml_traning.py

LucianoPC committed:
Update README

- Add how to run machine learning training
- Update how to run prepare data script
